wandb/cli/cli.py

```diff
@@ -954,6 +954,9 @@ def _user_args_to_dict(arguments, argument_type="P"):
     "W&B downloads artifacts from distributed URIs passed to parameters of "
     "type 'path' to subdirectories of storage_dir.",
 )
+@click.option(
+    "--uid", "-u", default=None, help="uid to use for the user, defaults to users uid",
+)
```

Comment:
tbh idk how i feel about having this in the args it feels too low level/specific to local. maybe something to specify at the config file level? the same way we'd specify eg anyscale details

Reply:
all right removing, and we'll have a the uid be specifyable in the config file under the docker key